.vscode/tasks.json

```diff
@@ -7,7 +7,7 @@
             "label": "watch",
             "type": "npm",
             "script": "watch",
-            "problemMatcher": "$tsc-watch",
+            "problemMatcher": ["$tsc-watch", "$ts-webpack-watch"],
```

Comment:
The vscode docs mention this but I'm not sure if it's actually needed. Will remove for now.